components/UI/button/ContainedButton.tsx

```diff
@@ -11,6 +11,9 @@ interface Props {
   disabled?: boolean
   type?: "button" | "submit" | "reset"
   icon?: React.ReactElement<SvgIconProps>
+  sx?: SxProps
+  component?: any
+  children?: JSX.Element[] | JSX.Element
```

Comment:
I think you can use React.ReactNode here

Reply:
ok